### ISSUE TITLE
This component doesn't define main assets in bower.json.

### ISSUE THREAD

**Issue by brikstar** [angular-ui/bootstrap#2644]
This component doesn't define main assets in bower.json.

**Comment by goldcaddy77** [comment]
Hey, I just got confused by this, too.  If this shouldn't be consumed directly, why do the instructions say: 

```
$ bower install angular-ui-bootstrap
```

Shouldn't they say: 

```
$ bower install angular-bootstrap
```

I'm happy to make the doc updates if that's the case.

**Comment by goldcaddy77** [comment]
FYI: doc updates: https://github.com/angular-ui/bootstrap/pull/3521

@karianna - any interest in chatting about getting proper build tools and the bower / npm files into this repo - it's become pretty much a standard.  I would be more than happy to put in a PR, too.

**Comment by karianna** [comment]
I'll have to get the other maintainers involved (I'm more of the admin guy here). I've pinged them

**Comment by goldcaddy77** [comment]
:+1:

**Comment by wesleycho** [comment]
Closing as resolved since I merged #3521.
